devtools/cli/src/main/java/io/quarkus/cli/create/TargetGAVGroup.java

```diff
@@ -52,6 +58,13 @@ void projectGav() {
                     }
                 }
             }
+            if (artifactId != CreateProjectHelper.DEFAULT_ARTIFACT_ID && !OK_ID.matcher(artifactId).matches()) {
+                throw new TypeConversionException(String.format(BAD_IDENTIFIER, "artifact", artifactId));
```

Comment:
Should this be `artifactId` and `groupId`?

Reply:
y.. probably should.